app/src/main/java/com/wafflestudio/snutt2/views/logged_in/home/TableListViewModel.kt

```diff
@@ -64,6 +64,22 @@ class TableListViewModel @Inject constructor(
         return tableRepository.deleteTable(tableId)
     }
 
+    suspend fun deleteAndSwitchTable(tableId: String) {
+        if (currentTableRepository.currentTable.value?.id == tableId) {
+            val newTableId: String
+            val courseBook = CourseBookDto(tableMap.value[tableId]!!.semester, tableMap.value[tableId]!!.year)
```

Comment:
!! 있는 건 참을 수가 없어
siblingTables 구하는 건
```suggestion
            val tableToDelete = tableMap.value[tableId] ?: return
            val siblingTables = tableMap.value.values.filter { it.isSameCourseBook(tableToDelete) }
```
이렇게 간단하게 ㄱㄱ (isSameCourseBook는 위에서 말한 새로 만들 SimpleTableDto의 함수)